test/testplan/testplan.md

```diff
@@ -0,0 +1,361 @@
+
+One of the long-term goals is to grow Web Monetization adoption by a factor of 10, in the next 5 years (by December 2029) such that Web Monetization functionality is either native to the most widely used Web browsers, or readily available through browser extensions on Web browser stores and marketplaces.
+
+## Objectives
+
+1. Validating that, once installed, users can configure Web Monetization settings on the extension.
+2. Validating that once the extension is successfully connected to a digital wallet, a user is able to.
+3. For web monetized websites, verify that the monetization event gets triggered.
+
+## Dependencies
+
+### Digital Wallets
+
+Web Monetization-enabled digital wallets are provided by licensed service providers who are regulated by the laws of the countries in which they operate. They provide digital wallets that can send or receive Web Monetization payments.
+
+Wallet availability in a specific country, or availability in particular currency depends on the wallet provider.
+To learn more, visit a specific wallet provider’s website. Below are the available wallet providers:
+
+- [Fynbos](https://wallet.fynbos.app/wallet)
+- [GateHub](https://gatehub.net/mobile)
+
+## Testing
+
+### Tools and Resources
+
+The resources and tools that you will need to test Web Monetization depend on your point of view.
+The Web Monetization personas have different perspectives and needs for testing:
+
+Persona 2: A website owner that wants to receive WM payments.
+
+| People                                                                              | Hardware                                                          | Digital wallet                                                                  | Environment & software                                                                        |
+| :---------------------------------------------------------------------------------- | :---------------------------------------------------------------- | :------------------------------------------------------------------------------ | :-------------------------------------------------------------------------------------------- |
+| **Persona 1**: A website visitor that wants to pay websites that are web monetized. | Any PC with internet access.                                      | A Web Monetization enabled digital wallet from which payments will be made.     | Any operating system (OS). A chromium-based Web browser (e.g. Chrome, Edge, Brave) or Firefox |
+| **Persona 2**: A website owner that wants to receive WM payments.                   | Any device with internet access (e.g. PC, tablet, mobile device). | A Web Monetization enabled digital wallet into which payments will be received. | Any operating system (OS). Any Web browser                                                    |
+
+### Reporting Issues
+
+1. Use any screen recording application to record the steps and results of your testing.
```

Comment:
Could we/should we suggest and/or link to one?

Reply:
There was an example initially, that I second-guessed because there are many options, it can be so subjective, and I'm not entirely sure about security concerns, or if it matters whether the software is open source or not.
How essential do you think it is to have an example?